EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/configuration/set_up_a_new_language.md

# Set up a new language

!!! note

>     If a new SiteAccess is introduced it is important to add a new Policy for the anonymous user Role.

Reviewer: ```suggestion
    If you introduce a new SiteAccess, it is important that you add a new Policy for the anonymous user Role.
```

Reviewer 2: ```suggestion
    If you introduce a new SiteAccess, it is important that you add a new Policy for the anonymous User Role
```

    Allow login for the new SiteAccess:

    ![](../img/configuration.png)

## Configuration

> - `ezpublish.siteaccess.list` - general configuration for new SiteAccess

Reviewer: ```suggestion
- `ezpublish.siteaccess.list` - general configuration for the new SiteAccess
```

> - `ezpublish.siteaccess.groups.ezdemo_site_clean_group` - assign the new SiteAccess to general group `ezdemo_site_clean_group`

Reviewer: ```suggestion
- `ezpublish.siteaccess.groups.ezdemo_site_clean_group` - assign the new SiteAccess to the general group `ezdemo_site_clean_group`
```

- `ezpublish.siteaccess.match.Compound\LogicalAnd` - define the domain and language for this new SiteAccess, e.g.:

``` yaml
website_be_nl:
    matchers:
        Map\URI:
            nl: true
        Map\Host:
            %domain_website_be%: true
    match: website_be_nl
```

> Next, configure session name and used languages:

Reviewer: ```suggestion
Next, configure a session name and used languages:
```

``` yaml
website_be_nl:            
    session:
        name: eZSESSID
    languages:
        - dut-NL
```

> - `ezpublish.siteaccess.ses_admin.languages` - the new language must be enabled for admin SiteAccess.

Reviewer: Can you avoid passive voice in the two lines?
